### PR TITLE
Replaced MindShift to Mindshift

### LINKED ISSUE #3: Replace all occurrences of "MindShift" with "Mindshift"
## Example

https://github.com/Mindshift-Analytics/roicalculator/blob/e858a28451166b8ad832d5b6ee294077e05d06cd/ui.R#L145

### PR DESCRIPTION
**Pull request by dogakantarci**
closes #3

### CODE REVIEW

**Comment by bkowshik** [comment]
Thank you for these changes @dogakantarci

File: README.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ For support or further information, please contact:
 
 Bhargav Kowshik
 
-Email: [bhargav\@mindshiftapps.net](mailto:bhargav@mindshiftapps.net){.email}
\ No newline at end of file
+Email: [bhargav\@Mindshiftapps.net](mailto:bhargav@Mindshiftapps.net){.email}
```
Comment (bkowshik) [inline README.md:15]: @dogakantarci Email address is better in lowercase. Ex: `bhargav@mindshiftapps.net`

**Review by bkowshik — CHANGES_REQUESTED** [review]
Requesting one change to your PR. Please see my comments.

File: README.md (hunk 1 above)
Comment (sanjmeh) [reply README.md:15]: @dogakantarci hope you will send another PR with this change.

**Pull request closed without merging**